### PR TITLE
Update changelog for 5.13

### PR DESCRIPTION
**Pull request by dabria**
Signed-off-by: Melissa Page <73hl10n@gmail.com>

## What is this change?
Updates to the changelog for 5.13 - release date and version number

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,12 +5,13 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic
 Versioning](http://semver.org/spec/v2.0.0.html).
 
-## Unreleased
+## [5.13.0] - 2019-08-09-09
```
Comment (nikkixdev) [inline CHANGELOG.md]:
```suggestion
## [5.13.0] - 2019-08-09
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,12 +5,13 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
...
 
 ### Added
 - Silenced entries are now retrieved from the cache when determining if an event
 is silenced.
 - Added --disable-assets flag to sensu-agent.
+- Add HTTP and directory support to sensuctl create
```
Comment (nikkixdev) [inline CHANGELOG.md]:
```suggestion
- Add HTTP and directory support to `sensuctl create`.
```

File: CHANGELOG.md (hunk 1 above)
Comment (dabria) [reply CHANGELOG.md]: Whoops thanks!

**Comment by dabria** [comment]
I just rebased, sorry that was a bit funky

File: CHANGELOG.md (hunk 2 above)
Comment (nikkixdev) [reply CHANGELOG.md]: Actually this needs to be in 5.13.0 maybe this was a weird merge..

**Pull request merged**